lib/audited/auditor.rb

```diff
@@ -81,9 +85,19 @@ def has_associated_audits
       def default_ignored_attributes
         [primary_key, inheritance_column]
       end
+
+      def normalize_audited_options(options)
+        [:only, :except, :redacted].each do |opt|
+          options[opt] = Array.wrap(options[opt]).map(&:to_s)
+        end
+        self.audited_options = options
+      end
     end
 
     module AuditedInstanceMethods
+
+      REDACTED = '[REDACTED]'
```

Comment:
The hard-coded string replacement (and name of the feature) concerns me a little, because it's invalid for different field types (e.g. integer or JSON fields) and might not be correct for all users.

Would you be able to implement a more generic "munge" interface, that might let a user handle different types or fields or enter their own text per-field? Perhaps something as simple as a lambda that's passed a hash of attributes.